docs/samples/graph/switch.yaml

```diff
@@ -10,21 +10,35 @@ spec:
       name: custom
       env:
       - name: "target"
-        value: "blue"
+        value: "single-1"
 ---
 apiVersion: "serving.kserve.io/v1beta1"
 kind: "InferenceService"
 metadata:
-  name: "green"
+  name: "single-2"
 spec:
   predictor:
     minReplicas: 1
     containers:
     - image: iamlovingit/bgtest:latest
       name: custom
-      env: 
+      env:
+      - name: "target"
+        value: "single-2"
+---
+apiVersion: "serving.kserve.io/v1beta1"
+kind: "InferenceService"
+metadata:
+  name: "single-3"
+spec:
+  predictor:
+    minReplicas: 1
+    containers:
+    - image: iamlovingit/bgtest:latest
```

Comment:
Can you also commit the example code so we can recreate the image if needed

Reply:
Sure